src/operationsClient.ts

```diff
@@ -581,7 +591,9 @@ export class OperationsClientBuilder {
      * @param {Object=} opts.clientConfig - The customized config to build the call settings. See {@link gax.constructSettings} for the format.
      */
     this.operationsClient = opts => {
-      if (gaxGrpc.fallback) {
+      if (gaxGrpc.fallback === 'rest') {
```

Comment:
can we just do `if (gaxGrpc.fallback) { opts.fallback = gaxGrpc.fallback; }` to assign whatever value is there?